src/Analyzer.Utilities.UnitTests/JsonLineNumberResolverTests.cs

```diff
@@ -213,14 +231,16 @@ public class JsonLineNumberResolverTests
             new object[] { "resources[0].properties.somePath", new object[] { "resources", 0, "properties", "somePath" }, "Path matches original template exactly" },
             new object[] { "parameters.parameter1.maxValue", new object[] { "parameters", "parameter1" }, "Beginning of path matches original template parameters, but has missing property" },
             new object[] { "resources[0].anExpandedProperty", new object[] { "resources", 0 }, "Beginning of path matches original template resources array, but has missing property" },
-            new object[] { "resources[4].properties.anotherProperty", new object[] { "resources", 0, "properties", "anotherProperty" }, "Path is in a copied resource" },
-            new object[] { "resources[4].properties.missingProperty", new object[] { "resources", 0, "properties" }, "Path is in a copied resource and has missing property" },
+            new object[] { "resources[4].properties.anotherProperty", new object[] { "resources", 1, "properties", "anotherProperty" }, "Path is in a moved (original from copy loop) resource" },
+            new object[] { "resources[5].properties.anotherProperty", new object[] { "resources", 1, "properties", "anotherProperty" }, "Path is in a new (copied) resource" },
```

Comment:
```suggestion
            new object[] { "resources[5].properties.anotherProperty", new object[] { "resources", 1, "properties", "anotherProperty" }, "Path is in a new (copied/duplicated) resource" },
```

Reply:
I'm thinking it may be clearer if we use the word duplicated to distinguish from _original_ and _duplicate_ copy of the _copy_ resource since we refer to the concept as "copy resources" and we refer to copies when we embed them as a sub-resource too.

Reply:
Good suggestion :)